### ISSUE TITLE
Complaint webform breaks images

### ISSUE THREAD

**Issue by freddymartinez9** [lucyparsons/OpenOversight#124]
Officer images should be available when you click "that's the officer" but this is how it looks now. 
<img width="1160" alt="screen shot 2016-11-21 at 10 25 17" src="https://cloud.githubusercontent.com/assets/3100220/20490822/dd351c6e-afd4-11e6-8c36-c229281fb0f1.png">

**Comment by freddymartinez9** [comment]
I tried to make a small edit to see if the multiple div tags where causing this and they are not. At the bottom of the comment you can see the diff I tried that didn't work. This seems to be caused by the CSS that provides `<img class="img-responsive thumbnail">`. It is possible that the div tags are contributing because images can't (AFAIK) be wider than the div container. 
  
```
             <center><h3>Warning! <small>The following link will open an external site in a new browser tab:</small></h3></p>
 <a href="http://ipraportal.iprachicago.org/pls/htmldb/f?p=1503:12:2709570244836141" target="_blank"><button type="button" class="btn btn-info btn-lrg">File Complaint Online</button></a></center> 
-            </div>
-            <div class="col-xs-6 col-lg-4">
                 <center>
                 <img class="img-responsive thumbnail" src="{{ officer_image }}" alt="Officer face">
                 </center>
@@ -24,7 +22,7 @@
                 {% if officer_middle_initial %}{{ officer_middle_initial }}. {% endif %}
                 {{ officer_last_name.lower()|title }} </h4>
                 <h4>Officer badge number: #{{ officer_star }} </h4>
-            </div>
+                </center>
```

**Comment by freddymartinez9** [comment]
With [this](https://github.com/lucyparsons/OpenOversight/commit/1cd549e3c0f0517f44209cf45c719cca207eb9aa) commit I am able to show the officer looking like this. If that looks good @redshiftzero I will make the PR.  
<img width="851" alt="screen shot 2016-11-27 at 01 31 09" src="https://cloud.githubusercontent.com/assets/3100220/20646568/9cd1d2f4-b441-11e6-9408-72dba8547f87.png">

**Comment by freddymartinez9** [comment]
Unfortunately, this won't fix #98 ...

**Comment by redshiftzero** [comment]
Looks good, PR away
